.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: generic
+sudo: required
+
+services:
+  - docker
+
+before_install:
+  - docker build -t iot-lab-gateway .
```

Comment:
If this line really needed ?
the iot-lab-gateway-test could be based on the released image on docker hub ?

Reply:
If it's not needed the travis build should be a little bit faster

Reply:
Don't think it would work. We want to test the current committed state, not the state of the released image. 

Right now: If test fails, then either the image or the gateway_code is bad, and both are at the same commit
If you test using a released image: If test fails, then either the image or the gateway_code is bad, and they are not at the same commit

it'd be great if we could cache the docker layers from dependencies (that's what's very long), and I think there are features in travis to maybe do that https://docs.travis-ci.com/user/caching/